man/systemd.netdev.xml

```diff
@@ -897,8 +900,75 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>FooOverUDP=</varname></term>
+        <listitem>
+          <para>A boolean. Specifies whether <varname>FooOverUDP=</varname> tunnel is to be configured.
+          Defaults to false. For more detail information see
+          <ulink url="https://lwn.net/Articles/614348">Foo over UDP</ulink></para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>FOUDestinationPort=</varname></term>
+        <listitem>
+          <para>The <varname>FOUDestinationPort=</varname> specifies the UDP destination port for encapsulation.
+          This field is mandatory and is not set by default.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>FOUSourcePort=</varname></term>
+        <listitem>
+          <para>The <varname>FOUSourcePort=</varname> specifies the
+          UDP source port for encapsulation. Defaults to <varname>auto</varname>,
+          that is the source port for packets is left to the network stack to decide.</para>
+        </listitem>
+      </varlistentry>
     </variablelist>
   </refsect1>
+
+  <refsect1>
+    <title>[FooOverUDP] Section Options</title>
+
+    <para>The <literal>[FooOverUDP]</literal> section only applies for
+    netdevs of kind <literal>fou</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Protocol=</varname></term>
+        <listitem>
+          <para>The <varname>Protocol=</varname> specifies the protocol number of the
+          packets arriving at <varname>FooOverUDPPort=</varname>. This field is mandatory and is not set
+          by default. Valid range is 1-255.</para>
```

Comment:
When encap type is FOU, then protocol is mandatory. However, protocol cannot specified when GUE is used.